src/main/scala/com/amazon/deequ/anomalydetection/RateOfChangeStrategy.scala

```diff
@@ -14,91 +14,18 @@
  *
  */
 
-package com.amazon.deequ.anomalydetection
 
-import breeze.linalg.DenseVector
+package com.amazon.deequ.anomalydetection
 
 /**
-  * Detects anomalies based on the values' rate of change.
-  * The order of the difference can be set manually.
-  * If it is set to 0, this strategy acts like the [[SimpleThresholdStrategy]].
-  *
-  * RateOfChangeStrategy(Some(-10.0), Some(10.0), 1) for example
-  * calculates the first discrete difference
-  * and if some point's value changes by more than 10.0 in one timestep, it flags it as an anomaly.
-  *
-  * @param maxRateDecrease Upper bound of accepted decrease (lower bound of increase).
-  * @param maxRateIncrease Upper bound of accepted growth.
-  * @param order           Order of the calculated difference.
-  *                        Set to 1 it calculates the difference between two consecutive values.
+  * Provided for backwards compatibility.
+  * the old [[RateOfChangeStrategy]] actually detects absolute changes
+  * so it has been migrated to [[AbsoluteChangeStrategy]]
+  * use [[RelativeRateOfChangeStrategy]] if you want to
+  * detect relative changes to the previous values
   */
+@deprecated
```

Comment:
Please add a short description to the deprecation annotation, describing why we deprecate it and which other class should be used instead.

Reply:
done